tests/frompyobject.rs

```diff
@@ -39,9 +39,9 @@ where
     Ok(dict)
 }
 
-
 #[test]
 fn test_conversion() -> PyResult<()> {
+    pyo3::prepare_freethreaded_python();
```

Comment:
`pyo3` no longer initializes Python automatically (and I couldn't get the `auto-initialize` feature to work with tests), so initializing Python manually.